### DISCUSSION TITLE
Improvement of SimaPro csv Export Documentation

### DISCUSSION THREAD

**Discussion by pheidak** [brightway-lca/brightway-documentation#90] [Documentation Request]
Hi @michaelweinold and team,

the current documentation includes much information on import functions but I´m struggling to find information on export. I´m particularly interested in exporting a database as a SimaPro CSV file. I think I remember that the previous documentation described the export as SimaPro CSV as challenging but possible and included useful information.

Could you let me know if there is a recommended workflow or if any information on export will be published soon?

Appreciate your work! 
Thanks

**Comment by michaelweinold** [comment]
Hey, @pheidak! Thank you for getting in touch. The documentation related to some of the many export functions was last improved in March of 2023:

- https://github.com/brightway-lca/brightway-documentation/discussions/66
- https://github.com/brightway-lca/brightway2-io/pull/169

I would need to have a look if we missed the SimaPro export function - and how that could be improved in the short run. I'll be sure to raise it in the [next developer planning meeting](https://github.com/brightway-lca/brightway-documentation/discussions/31#discussioncomment-5517714) also.

In the meantime, if you want to contribute to the documentation, please check out [the current guide here](https://documentation.brightway.dev/en/latest/source/contributing/contributing.html).

**Reply by pheidak** [reply]
Hi @michaelweinold,
could you give me an update regarding the last developer planning meeting? We as a team would love to contribute, we just don´t really know how to get started and are not very experienced in coding. We currently ask ourselves if a code block for Sima Pro CSV actually exists and just lacks documentation, or if the code needs to be written first. Could you answer this to me?
Thanks!

**Reply by michaelweinold** [reply]
Hi! I merged the remaining pull requests related to the documentation recently (https://github.com/brightway-lca/brightway-documentation/issues/88). All documentation related to exporting from Brightway is now available at [docs.brightway.dev/api/bw2io/export/](https://docs.brightway.dev/en/latest/api/bw2io/export/index.html).

From what I can see, there is nothing specific to SimaPro. So if SimaPro has issues with [the generic csv export already implemented](https://docs.brightway.dev/en/latest/api/bw2io/export/csv/index.html), you would need to write the code yourself.

**Reply by pheidak** [reply]
Alright, thank you for the update, that´s helpful!
